crates/sp-domains/src/lib.rs

```diff
@@ -282,6 +282,8 @@ pub struct ProofOfElection<DomainHash> {
     pub vrf_output: [u8; VRF_OUTPUT_LENGTH],
     /// VRF proof.
     pub vrf_proof: [u8; VRF_PROOF_LENGTH],
+    /// VRF signature hash.
+    pub vrf_signature_hash: Blake2b256Hash,
```

Comment:
Is `VRFSignature` serialized as `vrf_output || vrf_proof` (`||` as concatenation)? I would like to avoid putting an extra field `vrf_signature_hash` into proof of election and have the verifier derive and check it themselves. If there is no other way you can use `Hash(vrf_output || vrf_proof)` instead of `Hash(VRFSignature)`.

Reply:
makes sense, removed it (it was convenience anyways)